src/garage/torch/modules/multi_headed_mlp_module.py

```diff
@@ -22,28 +24,32 @@ class MultiHeadedMLPModule(nn.Module):
         hidden_b_init (callable): Initializer function for the bias
             of intermediate dense layer(s). The function should return a
             torch.Tensor.
-        output_nonlinearity (callable, list, or tuple): Activation function for
-            output dense layer. It should return a torch.Tensor.
+        output_nonlinearities (callable or list or tuple): Activation function
+            for output dense layer. It should return a torch.Tensor.
             Set it to None to maintain a linear activation.
             Size of the parameter should be 1 or equal to n_head
-        output_w_init (callable, list, or tuple): Initializer function for the
-            weight of output dense layer(s). The function should return a
+        output_w_inits (callable or list or tuple): Initializer function for
+            the weight of output dense layer(s). The function should return a
             torch.Tensor. Size of the parameter should be 1 or equal to n_head
-        output_b_init (callable, list, or tuple): Initializer function for the
-            bias of output dense layer(s). The function should return a
+        output_b_inits (callable or list or tuple): Initializer function for
+            the bias of output dense layer(s). The function should return a
             torch.Tensor. Size of the parameter should be 1 or equal to n_head
         layer_normalization (bool): Bool for using layer normalization or not.
 
     Return:
         The list of output torch. Tensor of the MLP
+
+    Raises:
+        ValueError: n_head is equal to 1
```

Comment:
is this really an error? what if I am doing some hyperparameter search with several numbers of heads, and 1 is one of the cases?

Reply:
if you want, you can change MLPModule to:

```python
class MLPModule(MultiHeadedMLPModule):

    def __init__(self, *args, **kwargs):
        super().__init__(1, *args, **kwargs)
````

Reply:
I think that is a good way!

Reply:
fixed